src/content/contribute/writing-a-scaffold.md

```diff
@@ -0,0 +1,392 @@
+---
+title: Writing a Scaffold
+sort: 3
+contributors:
+  - pranshuchittora
```

Comment:
I think that would be nice to also give credits to the people who previously worked on those files by adding them to the contributors in the heading. For example, I would add Even here that wrote the original tutorial :)

Reply:
@misterdev  :+1: